<?xml version="1.0"?>
<launch>=  
    <arg name="robot_name" default="my_gen3"/>
    <!-- Relaxed IK solver rate [Hz]. -->
    <arg name="solver_rate" default="750"/>


    <!-- Start relaxed ik -->
    <node pkg="relaxed_ik_ros1" type="relaxed_ik_rust.py" name="relaxed_ik" output="screen">
        <param name="robot_name" value="$(arg robot_name)"/>
        <param name="solver_rate" value="$(arg solver_rate)"/>
    </node>
</launch>
